packages/language-server/src/lib/documents/configLoader.ts

```diff
@@ -83,12 +85,19 @@ export class ConfigLoader {
         Logger.log('Trying to load configs for', directory);
 
         try {
-            const pathResults = this.globSync('**/svelte.config.{js,cjs,mjs}', {
-                cwd: directory,
-                // the second pattern is necessary because else fast-glob treats .tmp/../node_modules/.. as a valid match for some reason
-                ignore: ['**/node_modules/**', '**/.*/**'],
-                onlyFiles: true
-            });
+            const pathResults = new this.globSync({})
+                .withPathSeparator('/')
+                .exclude((_, path) => {
+                    // no / at the start, path could start with node_modules
+                    return path.includes('node_modules/') || path.includes('/.');
```

Comment:
It's necessary to not traverse into directories such as `.git` etc (directories starting with a dot are generally considered to be private), and I can't think of a situation where you would have a `svelte.config.js` in one of those directories - do you have such a case or why do you ask=

Reply:
I have used `.` directories for caching/generated content but don't think they needed checking/processing so far.
Was just wondering if this could be simplified, but it's probably also faster this way.